client/src/plus/ai-help/index.tsx

```diff
@@ -219,6 +220,56 @@ export function AIHelpInner() {
                             // eslint-disable-next-line jsx-a11y/anchor-has-content
                             return <a {...props} />;
                           },
+                          pre: ({ node, className, children, ...props }) => {
+                            let code = Children.toArray(children)
```

Comment:
Avoid let on variable that isn't modified:

```suggestion
                            const code = Children.toArray(children)
```